examples/dynamic-bridge-app/linux/main.cpp

```diff
@@ -184,6 +332,73 @@ bool emberAfActionsClusterInstantActionCallback(app::CommandHandler * commandObj
     return true;
 }
 
+EmberAfStatus emberAfExternalAttributeReadCallback(EndpointId endpoint, ClusterId clusterId,
+                                                   const EmberAfAttributeMetadata * attributeMetadata, uint8_t * buffer,
+                                                   uint16_t maxReadLength)
+{
+    uint16_t endpointIndex = emberAfGetDynamicIndexFromEndpoint(endpoint);
+
+    if ((endpointIndex >= CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT) || (gDevices[endpointIndex] == nullptr))
+        return EMBER_ZCL_STATUS_FAILURE;
+
+    chip::app::AttributeAccessInterface * accessInterface = chip::app::GetAttributeAccessOverride(endpoint, clusterId);
+
+    if (accessInterface == nullptr)
+        return EMBER_ZCL_STATUS_FAILURE;
+
+    // adding 64 bytes as padding to include the staging buffer used by
+    // TLVWriter, which is 17 bytes
+    std::vector<uint8_t> data(attributeMetadata->size + 64);
+
+    // read the attribute and write it to `data`
+    chip::TLV::TLVWriter writer;
+    writer.Init(data.data(), data.size());
+    CHIP_ERROR err = static_cast<CommonAttributeAccessInterface *>(accessInterface)
+                         ->Read(chip::app::ConcreteDataAttributePath(endpoint, clusterId, attributeMetadata->attributeId), writer);
+    if (err != CHIP_NO_ERROR)
+        return EMBER_ZCL_STATUS_FAILURE;
+    writer.Finalize();
+    data.resize(writer.GetLengthWritten());
+
+    // read from `data` and write to `buffer`
+    ReadOrWriteBufferForType(&data, buffer, attributeMetadata->attributeType, maxReadLength, false);
+
+    return EMBER_ZCL_STATUS_SUCCESS;
+}
+
+EmberAfStatus emberAfExternalAttributeWriteCallback(EndpointId endpoint, ClusterId clusterId,
+                                                    const EmberAfAttributeMetadata * attributeMetadata, uint8_t * buffer)
+{
+    uint16_t endpointIndex = emberAfGetDynamicIndexFromEndpoint(endpoint);
+
+    if ((endpointIndex >= CHIP_DEVICE_CONFIG_DYNAMIC_ENDPOINT_COUNT) || (gDevices[endpointIndex] == nullptr))
```

Comment:
please add `{}` for if statements even if one line, throughout the PR.

Reply:
Updated in [52d72b1](https://github.com/project-chip/connectedhomeip/pull/23840/commits/52d72b12fb91d58ad591d9edcb99f6c123e09c74)